app/views/local_transaction/search.html.erb

```diff
@@ -29,14 +29,14 @@
       text: "What you need to know",
       margin_bottom: 4
     } %>
-    <%= render "govuk_publishing_components/components/govspeak", {} do %>
-      <%= sanitize(@publication.need_to_know) %>
-    <% end %>
+    <%= render "govuk_publishing_components/components/govspeak", {
+      content: @publication.need_to_know.html_safe
+    } %>
   <% end %>
     <div class="more">
-      <%= render "govuk_publishing_components/components/govspeak", {} do %>
-        <%= sanitize(@publication.more_information) %>
-      <% end %>
+      <%= render "govuk_publishing_components/components/govspeak", {
+        content: @publication.more_information.html_safe
+      } %>
     </div>
   </section>
```

Comment:
As previous comment, none of these seem to be required fields in https://github.com/alphagov/govuk-content-schemas/blob/7d63f4f503c161e7756a867c02b58af4938dde1b/formats/local_transaction.jsonnet#L7-L10 so should probably put their outer containers within conditionals (this may be a theme for this review)

Reply:
Thanks @kevindew - will fix.